emu/processes/__init__.py

```diff
@@ -16,6 +16,7 @@
 from .wps_ncmeta import NCMeta
 from .wps_nonpyid import NonPyID
 from .wps_dry_run import SimpleDryRun
+from .wps_translation import Translation
```

Comment:
the `wps_translation` module is missing in this PR.

Reply:
Oups ! Added.